subprojects/testing-jvm/src/main/java/org/gradle/api/internal/tasks/testing/junit/JUnitTestClassProcessor.java

```diff
@@ -29,11 +30,11 @@
 
 public class JUnitTestClassProcessor implements TestClassProcessor {
     private static final Logger LOGGER = LoggerFactory.getLogger(JUnitTestClassProcessor.class);
-    private final IdGenerator<?> idGenerator;
+    protected final IdGenerator<?> idGenerator;
     private final ActorFactory actorFactory;
-    private final Clock clock;
+    protected final Clock clock;
```

Comment:
`protected` fields should be before `private`

Reply:
Thanks, done.